modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2362,11 +2362,9 @@ export class Wallet implements IWallet {
     } else {
       switch (this._wallet.type) {
         case 'hot':
+        case 'cold':
```

Comment:
Need to update the jsdocs.

Reply:
We don't currently have anything in the upcoming [dev portal](https://developers.bitgo-dev.com/) on token-enablement transactions. Talked with Mike on Slack and it sounds like we maybe should. Will add a Jira to our epic, [BG-46930](https://bitgoinc.atlassian.net/browse/BG-46930), if such a topic is needed.

Reply:
Here's a ticket for that as well: [BG-55694](https://bitgoinc.atlassian.net/browse/BG-55694)

Reply:
@mmcshinsky-bitgo updated jsdocs